grid/oauth/refresh-token/maps/twitter.test.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import {
+  REFRESH_TOKEN_REDACTED_VALUE,
+  refreshTokenTest,
+} from './refresh-token';
+
+const input = {
+  refreshToken: process.env.TWITTER_REFRESH_TOKEN,
+  clientId: process.env.TWITTER_CLIENT_ID,
+  clientSecret: process.env.TWITTER_CLIENT_SECRET,
+};
+
+refreshTokenTest('twitter', input, result => {
+  if (result.refreshToken !== REFRESH_TOKEN_REDACTED_VALUE) {
+    console.log('Latest Twitter refresh token:');
```

Comment:
Why do we need to log refresh token here?

Reply:
In case you want to rerun the recording, you need to paste the latest refresh token manually. This is a crude solution but IMO it's better than nothing.